src/linux/install-tools.sh

```diff
@@ -91,22 +106,46 @@ if [[ $1 != "-y" ]]; then
   print_prompt
 fi
 
-OS=$(uname -s)
-
-case $OS in
-  Linux)
-    if [[ -f /etc/redhat-release ]] ; then
-      install_redhat
-    elif [[ -f /etc/debian_version ]] ; then
-      install_debian
-    else
-      fatal "Sorry. Your OS ($OS) is not supported by this installer"
-    fi
-    ;;
-  *)
+if [[ "$(uname -s)" != "Linux" ]]
+then
     fatal "Sorry. Your OS ($OS) is not supported by this installer"
-    ;;
-esac
+fi
+
+if [[ -f /etc/os-release ]]
+then
+  . /etc/os-release
+
+  method=unknown
+
+  for distribution in ${ID_LIKE} ${ID}
+  do
+    case ${distribution} in
+      debian|ubuntu)           method=debian;;
+      fedora|centos|rhel)      method=redhat;;
+      suse|opensuse|sles|sled) method=suse;;
+    esac
+  done
```

Comment:
Mmm, was that meant to format differently? I'd expect indentation within the `case`, no?